portal_pages/models.py

```diff
@@ -208,7 +208,7 @@ class HomePage(Page):
 
 class HomePageHeroImageItem(Orderable, models.Model):
     home_page = ParentalKey(HomePage, related_name='hero_items')
-    blurb = RichTextField()
+    blurb = models.CharField(max_length=255)
```

Comment:
It looks like this change requires a data migration as well (to strip HTML).  My hero images are now rendering with HTML elements in them:

![rich-text](https://cloud.githubusercontent.com/assets/281961/8455565/a682102e-1fd3-11e5-8965-d33de8553362.png)